docs/docs/event-history/README.md

```diff
@@ -107,8 +107,14 @@ You'll need to replay an event to execute the workflow with the latest changes.
 
 ::: tip Beta access
 
-During the beta, paid accounts will have access to view the past 7 days of events across all of their workflows. This limit will increase as this trial continues.
+During the beta, all accounts will have access to view the past 7 days of events across all of their workflows. This limit will increase as this trial continues.
```

Comment:
```suggestion
During the beta, all accounts have access to view the past 7 days of events across all of their workflows. This limit increases as this trial continues.
```

Reply:
Changed "will have" to "have" and "will increase" to "increases" to follow Google's style guide on future tense.